src/index.js

```diff
@@ -72,6 +72,8 @@ class HMIApp extends React.Component {
         this.onTouchMove = this.onTouchMove.bind(this);
         this.onTouchEnd = this.onTouchEnd.bind(this);
         this.onTouchEvent = this.onTouchEvent.bind(this);
+        //setting default value
+        store.dispatch(resetTimeout());
```

Comment:
@LiliiaShlikhtLuxoft Seems [here](https://github.com/LuxoftSDL/generic_hmi/blob/d0f7a4f3874f095994e6955b13e6fc642af97aaa/src/js/reducers.js#L798) is a place for default initialization

Reply:
@ValeriiMalkov without dispatch resetTimeout will not appear.
You can find the same logic for ptuWithModemEnabled [here](https://github.com/LuxoftSDL/generic_hmi/blob/d0f7a4f3874f095994e6955b13e6fc642af97aaa/src/index.js#L88)

Reply:
This is not a default initilialization. This method will called after toggle checkbox [here](https://github.com/LuxoftSDL/generic_hmi/blob/d0f7a4f3874f095994e6955b13e6fc642af97aaa/src/index.js#L233)
This is not the same logic

Reply:
@ValeriiMalkov fixed [here](https://github.com/LuxoftSDL/generic_hmi/pull/28/commits/6ec70423976c8a3efe487a723e0f22195e5241ff)